client/auth/index.js

```diff
@@ -9,10 +9,12 @@ import page from 'page';
 import config from 'config';
 import controller from './controller';
 
+import { makeLayout, render as clientRender } from 'controller';
+
 module.exports = function() {
 	if ( config.isEnabled( 'oauth' ) ) {
-		page( '/login', controller.login );
-		page( '/authorize', controller.authorize );
-		page( '/api/oauth/token', controller.getToken );
+		page('/login', controller.login, makeLayout, clientRender);
```

Comment:
I'm wondering if we could take a bit different transformation instead. I noticed that we append `makeLayout, clientRender` to all `page` calls in `client` folder. What if we would create helper function that would do it behind the scenes? There is already similar method present: https://github.com/Automattic/wp-calypso/blob/361aa5843e4e3e7a62d21e1e65516dfa0bda8e28/client/controller/index.web.js#L66. We would need something like:

export function wrappedPage( route, ...middlewares ) {
	page(
		route,
		...middlewares,
		makeLayout,
		clientRender
	);
}